ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_node_type.schema.yml

```
@@ -753,14 +753,32 @@ $defs:
             documentation_options:
               table: node-type-l2-mlag-configuration
             type: str
+          mlag_peer_address_family:
+            documentation_options:
+              table: node-type-l2-mlag-configuration
+            description: |
+              Toggle to override the default (ipv4) address family used to establish MLAG Peer Link (control link).
+              Required when using IPv6 or other non-default values for MLAG leafs present in topology.
+              Requires minimum EOS version of 4.31.1f or higher.
```

Not sure what you mean with `other non-default values`?
```suggestion
            description: |-
              IP address family used to establish MLAG Peer Link (control link).
              `ipv6` requires EOS version 4.31.1F or higher.
              Note: `ipv6` is not supported in combination with a common MLAG peer link VLAN (ex. `mlag_l3_peer_vlan` set to 4094).
```

---

Manually added in suggestion.